index.html

```diff
@@ -0,0 +1,441 @@
+          >
+            Get my resume
+          </a>
+          <img
+            class="about-me-icon mobile-icon"
+            src="./media/svg/icon-2.svg"
+            alt="Icon"
+          />
+          <img
+            class="about-me-icon desktop-icon"
+            src="./media/svg/desktop-icon-2.svg"
+            alt="Icon"
+          />
+        </section>
+        <hr />
+        <div class="stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mwkakrqj" method="POST">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            maxlength="30"
+          />
+          <div class="name-form">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="First name"
+              maxlength="30"
+            />
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              placeholder="Last name"
+              maxlength="30"
+            />
+          </div>
+          <input
+            id="email"
+            name="_replyto"
+            placeholder="Email address"
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+          <button type="submit" class="link centered-link">Send</button>
+        </form>
```

Comment:
- [ ] Right now, the form only requires a message field. Kindly add requirements for names and email inputs as well since that is mandatory on your requirement list.